### PR TITLE
sot-integrity: explicit verdict precedence and materiality conditions

### PR DESCRIPTION
**Pull request by uda-lab-agent**
## Summary

Closes #142 (child of #127). Scope: `skills/sot-integrity/SKILL.md` only.

- Replaces "use the strictest verdict that fits" with an explicit first-match **Decision order**: `BROKEN` → `CONFLICTED` → `TRUSTED_WITH_GAPS` → `TRUSTED`, gated by one shared **materiality** definition ("can change what a downstream agent implements, validates, or authorizes within scope").
- Fixes the #128-style verdict-shadowing pattern: a document with *both* gaps and a bounded conflict now resolves deterministically to `CONFLICTED` (evaluated before `TRUSTED_WITH_GAPS`), never left ambiguous. New `Boundary Cases` section makes this explicit and shows the reachability of every verdict.
- Consolidates the five overlapping review lenses (`Authority quality`, `Source quality`, `Scope of trust`, `Conflict detection`, `Actionability under failure`) into four canonical **Audit dimensions**; output sections B/C/D now reference these findings instead of re-deriving criteria. `Actionability under failure` folded into the verdict table's agent-guidance column and Working rules (no content lost).
- Added a `Responsibility boundary` section aligned word-for-word in spirit with the post-#129 `metaplan` text: `metaplan` checks whether a plan names its governing SoT artifact, authority, and precedence; `sot-integrity` audits the artifact's own trustworthiness and is invoked by `metaplan` when that trust is material to execution readiness.
- Added a `Quality check` gate (mirrors the pattern already used in `metaplan` / `research-significance`) so the agent verifies precedence was applied top-down before finishing.

## Semantic changes vs. compression

**Semantic changes** (new behavior, not just wording):
- Explicit precedence order replaces "strictest verdict" — this changes how a mixed gaps+conflict document is classified (previously ambiguous/subjective → now deterministically `CONFLICTED`).
- New materiality gate: cosmetic/out-of-scope issues explicitly excluded from verdict computation (was implicit before).
- New `Boundary Cases` section (validation-by-construction): every verdict, every adjacent boundary, the mixed case, insufficient-information handling, and exact enum formatting.
- New `Responsibility boundary` section stating the `metaplan` ↔ `sot-integrity` division explicitly (previously only implied via the closing "Relationship to other skills" list).
- New `Quality check` self-verification gate.

**Compression only** (no behavior change):
- 5 review lenses → 4 audit dimensions (dropped "Actionability under failure" as a standalone lens; its content already lives in the verdict table + Working rules, nothing removed).
- `Use when` / `Do not use` bullets condensed.
- Output sections B/C/D shortened by referencing Audit dimensions instead of restating lens questions.
- Removed one duplicated "do not soften a verdict" sentence (kept once, in Working rules).

Preserved unchanged: frontmatter `description`, the full A–G output contract and its exact `Next action` enum, the `BROKEN` failure semantics ("Stop... downgrade to background context"), and the `Do not use this skill for...` exceptions.

## Counts

| | Before (origin/main) | After | Delta |
|---|---|---|---|
| Lines | 199 | 189 | −10 (−5%) |
| Words | 1342 | 1653 | +311 (+23%) |
| Chars | 8506 | 11056 | +2550 |
| Approx tokens (chars/4) | 2126 | 2764 | +638 (+30%) |

**Documented reason for the size increase:** issue #142's core ask (explicit decision order + materiality conditions + compact validation cases + an explicit `metaplan` boundary section) is net-new required content, not restatement. The compression work (lens consolidation, bullet condensing, de-duplication) is included and measurable, but it is smaller in magnitude than the required semantic addition. Per the #127 rewrite constraints, a shorter file is not the goal if it would hide a necessary exception or leave a verdict's precedence subjective — this rewrite prioritizes determinism (the literal defect being fixed) over raw size reduction, while still cutting genuine duplication.

## Validation performed

- `iconv -f UTF-8 -t UTF-8 skills/sot-integrity/SKILL.md` → valid UTF-8, no mangled characters.
- `wc -l -w` before (via `git show origin/main:...`) and after — table above.
- Enum-exact spot check: `Next action: proceed with caveats` (Boundary Cases) matches the `G. Next action` enum list character-for-character; no leading bullet/numbering.
- Mixed gaps+conflicts case traced through Decision order by hand: `BROKEN` ruled out → `CONFLICTED` matches first (bounded conflict present) → verdict is `CONFLICTED`, `TRUSTED_WITH_GAPS` is never reached for this input. All four verdicts confirmed reachable (no shadowing).
- `metaplan` boundary cross-checked against `skills/metaplan/SKILL.md` (post-#129, commit 2af9a08, base of this branch): metaplan's own `## Responsibility boundary` section says "If the trustworthiness of that artifact itself must be audited, invoke `sot-integrity`; do not duplicate its full trust model here" — this PR's `Responsibility boundary` section states the converse framing consistently, no contradiction.
- Confirmed only `skills/sot-integrity/SKILL.md` is touched (`git diff --stat` against origin/main shows a single file).

## Test plan

- [x] File is valid UTF-8 with no encoding artifacts
- [x] Every declared verdict (`TRUSTED`, `TRUSTED_WITH_GAPS`, `CONFLICTED`, `BROKEN`) has a Boundary Case and is reachable via Decision order
- [x] Each adjacent verdict boundary is covered (TRUSTED/TRUSTED_WITH_GAPS, TRUSTED_WITH_GAPS/CONFLICTED, CONFLICTED/BROKEN)
- [x] Mixed gaps+conflicts case resolves to exactly one verdict (`CONFLICTED`), documented and traced
- [x] Insufficient-information case has an explicit fallback (cap at `TRUSTED_WITH_GAPS`, name the missing input as a gap)
- [x] `Next action` enum formatting example matches the contract exactly
- [x] `metaplan` ↔ `sot-integrity` responsibility boundary is mutually consistent with post-#129 `skills/metaplan/SKILL.md`
- [x] Frontmatter description, A–G output contract, exceptions, and `BROKEN`/`stop` failure semantics preserved
- [x] Before/after line/word/token counts reported with documented reason for size change

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_016EpNbhDshcvbWPwBToXxXZ

### CODE REVIEW

**Comment by t-uda** [comment]
@codex please review this PR

File: skills/sot-integrity/SKILL.md [hunk 1]
```diff
@@ -157,42 +137,52 @@ Propose the smallest concrete edits that move the artifact toward `TRUSTED`. Pre
...
+- `TRUSTED` / `TRUSTED_WITH_GAPS` boundary: any material claim not yet grounded — however small — forces `TRUSTED_WITH_GAPS`; there is no rounding up.
+- `TRUSTED_WITH_GAPS` / `CONFLICTED` boundary: an unresolved gap stays `TRUSTED_WITH_GAPS` until it is confirmed to disagree with a companion source, at which point it becomes a conflict.
+- `CONFLICTED` / `BROKEN` boundary: `CONFLICTED` holds while at least one section outside the conflict's blast radius stays independently reliable; once the conflict undermines the artifact's core authority claim or spans all in-scope sections, it is `BROKEN`.
+- Mixed gaps+conflicts: a document with both a material conflict and unrelated gaps is `CONFLICTED` — apply Decision order top-down; never pick the softer verdict because gaps also exist.
+- Insufficient information: if `scope` or `companion_sources` needed to judge materiality are missing, request them; if a provisional read is unavoidable, cap the verdict at `TRUSTED_WITH_GAPS` and name the missing input itself as a gap.
+- Enum formatting: the final line reads exactly `Next action: proceed with caveats` — no leading bullet, numbering, or trailing commentary.
```
[the remark this answers is not in the document]
Comment (uda-lab-agent) [reply skills/sot-integrity/SKILL.md]: Fixed in dc79824. Section G now has an explicit verdict-to-next-action mapping (TRUSTED → proceed; TRUSTED_WITH_GAPS → proceed with caveats; CONFLICTED → repair or downgrade; BROKEN → stop or downgrade), and the Boundary Cases enum-formatting line is rephrased as a verdict-tied example ("for example, a TRUSTED_WITH_GAPS audit ends ...") instead of an unconditional exact line.

**Comment by uda-lab-agent** [comment]
Merged as eb4d792 (squash, branch deleted).

Gate evidence (§8):
- Clause 1: formal Review event id 4679045536 by chatgpt-codex-connector[bot], state COMMENTED, on commit 80e8849.
- Findings addressed: 1 substantive inline finding (P2, enum-formatting boundary case could misdirect the next action for non-TRUSTED_WITH_GAPS verdicts) — fixed in dc79824 with an explicit verdict-to-next-action mapping in section G; thread replied to and resolved.
- Open, not draft; no CI checks configured (empty = pass); no blocking labels; 0 unresolved threads; 0 unchecked boxes.
